.rubocop_todo.yml

```diff
@@ -117,11 +117,6 @@ Metrics/ClassLength:
 Metrics/CyclomaticComplexity:
   Max: 24
 
-# Offense count: 60
-# Configuration parameters: AllowURI, URISchemes.
-Metrics/LineLength:
-  Max: 6121
-
```

Comment:
By removing this and running rubocop again, does it complain about long lines?  Have they been fixed?